app/views/conference/_sponsors.html.haml

```diff
@@ -6,28 +6,17 @@
   .row
     .col-md-12
       %h2 Sponsors
-      %p.lead
-        whom without,
-        %span.notranslate
-          = @conference.short_title
-        wouldn't be possible! 
   - @conference.sponsorship_levels.each do |sponsorship_level|
     -if sponsorship_level.sponsors.any?
+      %h4
+        = sponsorship_level.title
       - sponsorship_level.sponsors.each_slice(3) do |slice|
-        .row.row-centered
+        .row
           - slice.each do |sponsor|
-            .col-md-4.col-sm-4.col-centered.col-top
-              .thumbnail
-                - if sponsor.logo.blank?
-                  %h3.text-center
-                    = link_to(sponsor.website_url, title: "#{sponsor.name}") do
-                      = sponsor.name
-                - else
-                  = link_to(sponsor.website_url, class: 'thumbnail') do
-                    = image_tag sponsor.logo, class: "img-responsive img-sponsor img-sponsor-#{sponsorship_level.position}", title: "#{sponsor.name}"
-                .caption
-                  -if sponsor.description.present?
-                    = markdown(sponsor.description)
+            .col-md-4.col-sm-4.col-top
+              - if sponsor.logo.present?
```

Comment:
validation is [added in this PR](https://github.com/openSUSE/osem/pull/951/files#diff-c50c4ba5add3c326d0f0646377bbbcd4R10) which ensures the presence of logo. (`- if ..` is not needed)